### ISSUE TITLE
How to split the dataset?

### ISSUE THREAD

**Issue by LeoGeralt** [libuyu/PhantomDanceDataset#5]
Thanks for sharing!
I'm interested in your dataset, and I want to know how to split this dataset. I haven't found any specific methods for dividing the training set and test set in both GitHub and the paper. Could you please provide the specific details of the division?
Looking forward to your reply!
